csrc/scheduler/normalization_inner_outer_utils.cpp

```diff
@@ -0,0 +1,312 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <expr_evaluator.h>
+#include <grouped_reduction.h>
+#include <id_model/id_model.h>
```

Comment:
I can't help but notice that we are including id_model.h but not using IdModel in this source.

A nitpick: can we clean up the includes?

Reply:
Good point. I removed unused header files.